pkgs/os-specific/linux/ch9344/default.nix

```diff
@@ -34,13 +35,12 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     homepage = "https://www.wch-ic.com/";
-    downloadPage = "https://www.wch.cn/downloads/CH9344SER_LINUX_ZIP.html";
+    downloadPage = "https://github.com/WCHSoftGroup/ch9344ser_linux";
     description = "WCH CH9344/CH348 UART driver";
     longDescription = ''
       A kernel module for WinChipHead CH9344/CH348 USB To Multi Serial Ports controller.
     '';
-    # Archive contains no license.
-    license = licenses.unfree;
+    license = licenses.gpl2;
```

Comment:
```suggestion
    license = licenses.gpl2Only;
```

Reply:
`gpl2` was chosen since that's the exact match to the header's SPDX ID, even though I knew it was deprecated.